### PR TITLE
Added wrapper for image memory allocate and image memory free

### PR DESCRIPTION
**Pull request by VinoshanT**
Signed-off-by: Vinoshan Tharmalenkam <Vinoshan.Tharmalenkam@ibm.com>

### CODE REVIEW

File: runtime/vm/JVMImage.cpp [hunk 1]
```diff
@@ -196,9 +196,22 @@ image_mem_allocate_memory(struct OMRPortLibrary* portLibrary, uintptr_t byteAmou
...
 	JVMImage *jvmImage = JVMImage::getInstance();
 	jvmImage->freeSubAllocatedMemory(memoryPointer);
-} 
+}
+
+extern "C" void
+img_mem_free_mem(void* memoryPointer){
+    JVMImage *jvmImage = JVMImage::getInstance();
+    image_mem_free_memory(jvmImage->getPortLibrary(), memoryPointer);
+}
```
Comment (Ershany-IBM) [inline runtime/vm/JVMImage.cpp]: Small thing, but make sure there is one newline character at the end of the file (on its own line)

File: runtime/vm/JVMImage.cpp [hunk 2]
```diff
@@ -196,9 +196,22 @@ image_mem_allocate_memory(struct OMRPortLibrary* portLibrary, uintptr_t byteAmou
...
 extern "C" void
 image_mem_free_memory(struct OMRPortLibrary* portLibrary, void* memoryPointer)
 {
 	JVMImage *jvmImage = JVMImage::getInstance();
 	jvmImage->freeSubAllocatedMemory(memoryPointer);
-} 
+}
+
+extern "C" void
+img_mem_free_mem(void* memoryPointer){
```
Comment (akshayben) [inline runtime/vm/JVMImage.cpp]: put the bracket on a new line

File: runtime/oti/jvmimage_api.h [hunk 3]
```diff
@@ -31,6 +31,15 @@
...
 #endif
 
+/*
+* Allocate memory in heap image
+* 
+* @param size[in] size to allocate
+*
+* returns pointer to allocated memory on success, NULL on failure 
+*/
+void* img_mem_allocate_mem(uintptr_t size);
```
Comment (akshayben) [inline runtime/oti/jvmimage_api.h]: byteAmount

Comment (akshayben) [reply runtime/oti/jvmimage_api.h]: Call this mem_allocate_memory and mem_free_memory (suggestion from [https://github.com/TobiAjila/openj9/pull/2]) Not too important

File: runtime/oti/jvmimage_api.h [hunk 4]
```diff
@@ -43,6 +52,13 @@ extern "C" {
 */
 void* image_mem_allocate_memory(struct OMRPortLibrary* portLibrary, uintptr_t byteAmount, const char* callSite, uint32_t category);
 
+/*
+* Free memory in heap image
+*
+* @param memoryPointer[in] pointer to address for free
+*/
+void mem_free_memory(void* memoryPointer);
```
Comment (akshayben) [inline runtime/oti/jvmimage_api.h]: void *memoryPointer

File: runtime/vm/JVMImage.cpp [hunk 5]
```diff
@@ -196,9 +196,23 @@ image_mem_allocate_memory(struct OMRPortLibrary* portLibrary, uintptr_t byteAmou
...
 extern "C" void
 image_mem_free_memory(struct OMRPortLibrary* portLibrary, void* memoryPointer)
 {
 	JVMImage *jvmImage = JVMImage::getInstance();
 	jvmImage->freeSubAllocatedMemory(memoryPointer);
-} 
+}
+
+extern "C" void
+mem_free_memory(void* memoryPointer)
```
Comment (akshayben) [inline runtime/vm/JVMImage.cpp]: same as above

**Review by akshayben — APPROVED** [review]

**Pull request merged**